rma/migrations/14.0.1.1.2/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2025 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+_field_renames = [
+    ("stock.move", "stock_move", "forced_lot_id", "restrict_lot_id"),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    cr = env.cr
+    for field in _field_renames:
+        if openupgrade.table_exists(cr, field[1]) and openupgrade.column_exists(
+            cr, field[1], field[2]
+        ):
+            openupgrade.rename_fields(env, [field])
```

Comment:
Please, make sure that this not override restrict_lot_id values if the script was already executed in a previous version (14.0 in this case)

Reply:
Updated!